script.js

```diff
@@ -1,4 +1,116 @@
+//-------------------------------------------Base Code---------------------------------------
+//array to store final number concatenated by the players
```

Comment:
Comment is not clear whether `diceRolls` is going to store rolls from a single player or for multiple players